packages/react-devtools-shared/src/devtools/views/Profiler/ProfilerContext.js

```diff
@@ -43,7 +43,7 @@ export type Context = {|
   // 1. The selected root in the Components tree (if it has any profiling data) or
   // 2. The first root in the list with profiling data.
   rootID: number | null,
-  setRootID: (id: number) => void,
+  setRootIDcleanFiber: (id: number) => void,
```

Comment:
I think `setRootIDcleanFiber` is kind of an awkward name. Prefer to just leave the name we pass down through the context as `setRootID` (and change the implementation to also clear the fiber ID like you did)